contracts/artifacts/DelegatedOptIn.approval.teal

```diff
@@ -0,0 +1,165 @@
+#pragma version 9
+
+txn ApplicationID
+int 0
+>
+int 6
+*
+txn OnCompletion
++
+switch create_NoOp NOT_IMPLEMENTED NOT_IMPLEMENTED NOT_IMPLEMENTED NOT_IMPLEMENTED NOT_IMPLEMENTED call_NoOp
```

Comment:
Yeah this is how TEAL handles routing. I eventually want to make this an ARC so explorers can just parse the first couple lines of TEAL and if its `txn ApplicationID; int 0; >; int 6; *; txn OnCompletion; +; switch ...` follow the branch for delete/update to see if it's `err`, which would allow for explorers to show a binary flag on whether a contract is mutable or not.

Reply:
Added a comment explaining what the code is doing and why in c5d72037dcc7c69de47a1e8a018707090a949b9e